extensions/wikia/AdEngine/AdEngine2Hooks.class.php

```diff
@@ -13,6 +13,7 @@ class AdEngine2Hooks {
 	const ASSET_GROUP_ADENGINE_TRACKING = 'adengine2_tracking_js';
 	const ASSET_GROUP_LIFTIUM = 'liftium_ads_js';
 	const ASSET_GROUP_LIFTIUM_EXTRA = 'liftium_ads_extra_js';
+	const ADS_STAGING_HEADER = 'externaltest';
```

Comment:
That's a very good question! :+1: 

It's connected to [VCL line](https://github.com/Wikia/wikia-vcl/blob/master/wikia.com/control-stage.vcl#L19) and after thinking about it again it'd be good if we write an integration test checking if `X-Staging` really equals `externaltest` for both aliases.